man/repart.d.xml

```diff
@@ -422,6 +422,22 @@
         <xi:include href="version-info.xml" xpointer="v247"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>MkfsOptions=</varname></term>
```

Comment:
So I hate the concept, but I have no better idea. It seems unrealistic trying to add an explicit option for each possibly mkfs option.

but please, let's make this a bit less cryptic, with fewer unnecessary abbreviations. And emphasize that this are *additional* options only. Maybe call this "MakeFileSystemOptionsExtra="?

Reply:
Will keep that in mind. Not applicable anymore as documentation has been removed.